modules/stitching/include/opencv2/stitching/detail/matchers.hpp

```diff
@@ -202,11 +212,12 @@ class CV_EXPORTS_W BestOf2NearestRangeMatcher : public BestOf2NearestMatcher
     CV_WRAP BestOf2NearestRangeMatcher(int range_width = 5, bool try_use_gpu = false, float match_conf = 0.3f,
                             int num_matches_thresh1 = 6, int num_matches_thresh2 = 6);
 
-    void operator ()(const std::vector<ImageFeatures> &features, std::vector<MatchesInfo> &pairwise_matches,
-                     const cv::UMat &mask = cv::UMat());
```

Comment:
We should not remove methods until 5.x.

Reply:
@alalek I'm not familiar with the rules of the project and versioning, you may be fully aware of this, but unless I am missing something the method is now correctly inherited from its superclass and works as you can see from https://github.com/opencv/opencv/pull/22329/files/26a7647e0e7c665e48e058f71cc1ab952cc4303d#diff-db221159aa572ef04177249f99bc952a3c8e11f8d34029118db5bbbbf787bc1eR134. I do not think the API changes unless I missed something. (Sorry if you knew this already!)

Reply:
@alalek The base class has the same method. It was overload. The PR brings the same strategy pattern as it was before and calls virtual method internally to resolve class specific behavior. With the new approach old code will work as designed. I do not expect changes for correct C++ code after relbuild with new OpenCV.

Reply:
updated compliance tool's exclude list.